test/e2e/api/users.test.ts

```diff
@@ -0,0 +1,66 @@
+import * as nock from 'nock';
+import * as request from 'supertest';
+import { Application } from 'express';
```

Comment:
I think it's unused. Please fix your tslint or ts => https://palantir.github.io/tslint/rules/no-unused-variable/